core/src/test/java/edu/ucr/cs/riple/core/ConfigurationTest.java

```diff
@@ -118,6 +118,7 @@ public void testRequiredFlagsMissingCli() {
               IllegalArgumentException.class,
               () -> new Config(makeCommandLineArguments(incompleteFlags)));
       // Check the error message.
+      System.out.println(ex.getMessage());
```

Comment:
Was this for debugging? I don't think the tests should print to stdout on every run, and we are already checking the message below on a successful run. Let's remove this line.

Reply:
Yes exactly, this is left out code from debugging. bd21c3d0e4abd6d8e120b7bc20523056e5187e0b